pkg/security/admission/admission_test.go

```diff
@@ -437,26 +438,40 @@ func TestAdmit(t *testing.T) {
 					t.Errorf("%s should have validated against %s but found %s", k, saSCC.Name, validatedSCC)
 				}
 
-				// ensure anything we expected to be defaulted on the container level is set
-				if *containers[0].SecurityContext.RunAsUser != v.expectedUID {
-					t.Errorf("%s expected UID %d but found %d", k, v.expectedUID, *containers[0].SecurityContext.RunAsUser)
+				podSC := securitycontext.NewPodSecurityContextAccessor(v.pod.Spec.SecurityContext)
```

Comment:
@liggitt I decided to use accessors here but I suspect there should be a better option for updating this test. WDYT?

Reply:
I think it needs to look like the upstream tests... expectedPodSecurityContext and expectedContainerSecurityContext, and do literal comparisons

Reply:
I've filled a separate PR for this change: https://github.com/openshift/origin/pull/17296